.github/workflows/ci-base-tests-linux.yml

```diff
@@ -51,3 +51,34 @@ jobs:
             --ignore=./smarts/env/tests/test_benchmark.py \
             --ignore=./examples/tests/test_learning.py \
             -k 'not test_long_determinism'
+
+  benchmark:
```

Comment:
How long does this test take to run, we would want it to be ~2 mins. Preferably less.